cmd/access/node_builder/access_node_builder.go

```diff
@@ -456,11 +589,22 @@ func (builder *FlowAccessNodeBuilder) extraFlags() {
 		flags.StringSliceVar(&builder.bootstrapNodePublicKeys, "bootstrap-node-public-keys", defaultConfig.bootstrapNodePublicKeys, "the networking public key of the bootstrap access node if this is an unstaked access node (in the same order as the bootstrap node addresses) e.g. \"d57a5e9c5.....\",\"44ded42d....\"")
 		flags.BoolVar(&builder.supportsUnstakedFollower, "supports-unstaked-node", defaultConfig.supportsUnstakedFollower, "true if this staked access node supports unstaked node")
 		flags.StringVar(&builder.PublicNetworkConfig.BindAddress, "public-network-address", defaultConfig.PublicNetworkConfig.BindAddress, "staked access node's public network bind address")
+		flags.BoolVar(&builder.executionDataSyncEnabled, "execution-data-sync-enabled", defaultConfig.executionDataSyncEnabled, "whether to enable the execution data sync protocol")
+		flags.BoolVar(&builder.executionDataCheckEnabled, "execution-data-startup-check", defaultConfig.executionDataSyncEnabled, "whether to check execution data exists for all heights during startup")
+		flags.DurationVar(&builder.executionDataFetchTimeout, "execution-data-fetch-timeout", defaultConfig.executionDataFetchTimeout, "timeout to use when fetching execution data from the network e.g. 300s")
+		flags.Uint64Var(&builder.executionDataStartBlockHeight, "execution-data-start-height", defaultConfig.executionDataStartBlockHeight, "height of first block to sync execution data from")
+		flags.Uint64Var(&builder.executionDataMaxCachedEntries, "execution-data-max-cache-entries", defaultConfig.executionDataMaxCachedEntries, "max number of execution data entries to cache for notifications")
+		flags.Uint64Var(&builder.executionDataMaxSearchAhead, "execution-data-max-search-ahead", defaultConfig.executionDataMaxSearchAhead, "max number of heights to search ahead of the lowest outstanding execution data height")
+		flags.StringVar(&builder.executionDataDir, "execution-data-dir", defaultConfig.executionDataDir, "directory to use for Execution Data database")
```

Comment:
most likely operators will want to specify a custom dir, that they mount into the container, but it could go in the internal volume if they have enough space allocated.